tests/e2e/e2e.go

```diff
@@ -363,143 +316,101 @@ func (test *TestCtx) VppRelease() string {
 }
 
 // GenericClient provides generic client for communication with default VPP-Agent test component
+// Deprecated: use ctx.Agent.GenericClient(...) instead
```

Comment:
I would not mark this as deprecated. Since most of the test cases use a single agent, using this shorter call for the main agent makes it much more easier to read keeps.